Src/BootCamp.Chapter1/Program.cs

```diff
@@ -1,9 +1,19 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+namespace BootCamp.Chapter1
 {
     class Program
     {
         static void Main(string[] args)
         {
+            var arr = new int[3];
+            arr[0] = 3;
+            arr[1] = 1;
+            //arr[2] = 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4954424)